packages/docusaurus-theme-redoc/src/theme/Redoc/styles.css

```diff
@@ -71,18 +71,14 @@ html:not([data-theme="dark"]) .redocusaurus div[id^="react-tabs"] > div:nth-chil
 
 /* ------ Schema Styling Overrides ------- */
 
-.redocusaurus table th, table td {
+.redocusaurus table th, .redocusaurus table td {
   border: none;
 }
 
-.redocusaurus table:not(.security-details) td {
```

Comment:
This removes the border-bottom of tables that are not part of `security-details`, is there a reason why this was added? The tables in `security-details` render fine without this